src/display/annotation_layer.js

```diff
@@ -885,7 +892,6 @@ class PopupElement {
     // disappear too. In that special case, hiding the wrapper suffices.
     this.hideElement = this.hideWrapper ? wrapper : this.container;
     this.hideElement.setAttribute("hidden", true);
-
```

Comment:
This change is unrelated, so please revert it (the whitespace here it intentional to separate the text annotation logic from the rest).